arch/arm64/mmu.c

```diff
@@ -30,6 +30,8 @@ pte_t arm64_kernel_translation_table[MMU_KERNEL_PAGE_TABLE_ENTRIES_TOP]
 __ALIGNED(MMU_KERNEL_PAGE_TABLE_ENTRIES_TOP * 8)
 __SECTION(".bss.prebss.translation_table");
 
+uint64_t arm64_mmu_tcr_flags __SECTION(".bss.prebss.tcr_flags");
```

Comment:
suggestion: comment mentioning that this is the base TCR flags, computed from early init code in start.S

Reply:
Done